Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Without looking to get into it with you, I'm not sure that argument scales really.

I'm as big an Apple fan as the next human willing to drop thousands on a laptop and not think _"Damn, what am I doing with my life choices"_, but I think a part of being such a fan of someone's work is also being willing to not sort of apply that regardless. It's not a diss on Apple to suggest that handling every little software update is not the top of their to-do list & this is partly why things like Homebrew and MacPorts exist & seemingly exist with Apple's blessing at that.

I mean if you flip your answer onto politics, I'm sure neither of us know as much about crafting legislation as our elected MPs and MSPs, but I know from prior conversations we've both got some strong views around various issues & how they should be handled. I don't think it kills anyone to acknowledge that even those with lesser experience in a field are allowed to have an opinion on how an issue should be handled, albeit ideally whilst remaining open to being proven wrong or having minds changed by new information or developments.